Not able to install @a6kme/math as a dependency

Library installation fails, since the `install` hook tries to run `jest` which is not present in the client
```
> @a6kme/math@1.0.1 install /Users/abhishekkumar/EarlySalary/WebJourney/esnextjswebjourney/node_modules/@a6kme/math
> npm run test


> @a6kme/math@1.0.1 test /Users/abhishekkumar/EarlySalary/WebJourney/esnextjswebjourney/node_modules/@a6kme/math
> jest

sh: jest: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! @a6kme/math@1.0.1 test: `jest`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the @a6kme/math@1.0.1 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/abhishekkumar/.npm/_logs/2019-04-17T11_16_42_094Z-debug.log
npm WARN jss-camel-case@6.1.0 requires a peer of jss@^9.7.0 but none is installed. You must install peer dependencies yourself.
npm WARN jss-global@3.0.0 requires a peer of jss@^9.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN jss-default-unit@8.0.2 requires a peer of jss@^9.4.0 but none is installed. You must install peer dependencies yourself.
npm WARN jss-nested@6.0.1 requires a peer of jss@^9.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN jss-props-sort@6.0.0 requires a peer of jss@^9.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN jss-vendor-prefixer@7.0.0 requires a peer of jss@^9.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN unistore@3.2.1 requires a peer of preact@* but none is installed. You must install peer dependencies yourself.
npm WARN ajv-keywords@3.4.0 requires a peer of ajv@^6.9.1 but none is installed. You must install peer dependencies yourself.
npm WARN jss-preset-default@4.5.0 requires a peer of jss@^9.7.0 but none is installed. You must install peer dependencies yourself.
npm WARN jss-compose@5.0.0 requires a peer of jss@^9.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN jss-expand@5.3.0 requires a peer of jss@^9.4.0 but none is installed. You must install peer dependencies yourself.
npm WARN jss-extend@6.2.0 requires a peer of jss@^9.7.0 but none is installed. You must install peer dependencies yourself.
npm WARN jss-template@1.0.1 requires a peer of jss@^9.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN es-next@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @a6kme/math@1.0.1 install: `npm run test`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @a6kme/math@1.0.1 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/abhishekkumar/.npm/_logs/2019-04-17T11_16_42_632Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@a6kme/math",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "description": "",
   "main": "dist/math.js",
   "scripts": {

--- a/package.json
+++ b/package.json
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "jest",
     "build": "webpack --mode=production",
-    "install": "npm run test",
+    "prepare": "npm run test",
     "posttest": "npm run build"
   },
   "repository": {

